helm/auto-deployment/bold-bi-auto-deployment-using-helm.md

```diff
@@ -0,0 +1,439 @@
+      </td>
+    </tr>
+    <tr>
+      <td>
+       siteName
+      </td>
+      <td>
+      This is organization name.     
+      <br />
+       If the value is not given, the site will be deployed using the default name.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       siteIdentifier
+      </td>
+      <td>     
+       This is site identifier, and it will be the part of the application URL.
+      <br />
+      If the value is not given, the site will be deployed using the default value.
+      </td>
+    </tr>
+</table>
+<br/>
+
+> **Note:** Items marked with `*` are mandatory fields in values.yaml
+
+<details>
+          <summary>
+            <span style="color:blue"> Example for above environment variables:</span>  
+          </summary>
+
+            # Environment Variables For Application Branding.
+            # This environment variables helps us to customize the branding.  
+
+            brandingDetails:
+
+              # This is the header logo for the application, and the preferred image size is 40 x 40 pixels.
+              main_logo:  https://i.postimg.cc/6pJTRr8W/autumn-fall-nature-scene-autumnal-park-beautiful-77869343-1.jpg
+
+              # This is the login logo for the application, and the preferred image size is 200 x 40 pixels.
+              login_logo:  https://i.postimg.cc/6pJTRr8W/autumn-fall-nature-scene-autumnal-park-beautiful-77869343-1.jpg
+
+              # This is an email logo, and the preferred image size is 200 x 40 pixels.
+              email_logo:  https://i.postimg.cc/6pJTRr8W/autumn-fall-nature-scene-autumnal-park-beautiful-77869343-1.jpg
+
+              # This is a favicon, and the preferred image size is 40 x 40 pixels.
+              favicon:  https://i.postimg.cc/6pJTRr8W/autumn-fall-nature-scene-autumnal-park-beautiful-77869343-1.jpg
+
+              # This is powered by the logo, and the preferred size is 100 x 25 pixels.  
+              footer_logo:  https://i.postimg.cc/6pJTRr8W/autumn-fall-nature-scene-autumnal-park-beautiful-77869343-1.jpg
+
+              # This is organization name. 
+              site_name: Autodeployment
+
+              # This is site identifier, and it will be the part of the application URL.
+              site_identifier: Branding
+</details>
+
+Run the following command to delpoy Bold BI in your cluster.
+
+```console
+helm install [RELEASE_NAME] boldbi/boldbi -f [Crafted values.yaml file]
+```
+Ex:  `helm install boldbi boldbi/boldbi -f my-values.yaml`
+
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
+
+## Upgrade
+
+Run the following command to get the latest version of Bold BI helm chart.
+
+```console
+helm repo update
+```
+
+Run the below command to apply changes in your Bold BI release or upgrading Bold BI to latest version.
```

Comment:
Run the following command to apply changes in your Bold BI release or upgrade Bold BI to the latest version.

Reply:
concern addressed